lib/ex_doc/cli.ex

```diff
@@ -63,15 +63,17 @@ defmodule ExDoc.CLI do
       PROJECT            Project name
       VERSION            Version number
       BEAMS              Path to compiled beam files
-      -o, --output       Path to output docs, default: docs
-      --readme           Path to README.md file to generate a project README, default: nil
-      -f, --formatter    Docs formatter to use; default: html
+      -o, --output       Path to output docs, default: "doc"
+      --readme           Path to README.md file to generate a project README, default: `nil`
+      -f, --formatter    Docs formatter to use, default: "html"
       -c, --config       Path to the formatter's config file
-      -r, --source-root  Path to the source code root, default: .
+      -r, --source-root  Path to the source code root, default: "."
       -u, --source-url   URL to the source code
-      --source-ref       Branch/commit/tag used for source link inference, default: master
-      -m, --main         The main, entry-point module in docs
+      --source-ref       Branch/commit/tag used for source link inference, default: "master"
+      -m, --main         The main, entry-point module in docs,
+                           default: "overview" when --fomatter is "html"
       -p  --homepage-url URL to link to for the site name
+      -i, --index        Index page, default: "index.html" when --fomatter is "html"
```

Comment:
I thought of  --index as a to store the address to the the index "index.htm", "", "./" or "/"
but --main as the path to what it should redirect to.
I think I broke my own rule of not linking.
I don't think we need need to ever link to index.html so i don't think we need config.index
the only thing we need is when people visit "test.com/doc/" it takes them to "overview.html" or whatever set by --main
What do you think